### PR TITLE
Making the release pages data driven

### PR DESCRIPTION
**Pull request by indraraj**
- Release version pages are now data-driven.

WIP
Displaying the correct date and summary on the release cards(release landing page)

### CODE REVIEW

**Review by uidoyen — APPROVED** [review]
Tested locally. Looking good.

File: _includes/release.html [hunk 1]
```diff
@@ -1,157 +1,188 @@
...
+      <div class="series raised">
+        <div class="series-banner">
+          <h4 class="series-label">
+            <div class="span label label-stable">stable</div>
+          </h4>
+        </div>
+        <div class="clear"></div>
+        <h3 class="version">{{ version }}</h3>
+        <h6 class="version-date">2020-09-24</h6>
+        <div class="description">Align code with Quarkus 1.9</div>
```
Comment (gunnarmorling) [inline _includes/release.html:17]: Those values should be obtainted from the data files, not be hard-coded.

File: _includes/release.html [hunk 2]
```diff
@@ -1,157 +1,188 @@
...
+          <div class="series raised">
+            <div class="series-banner">
+              <h4 class="series-label">
+                <div class="span label label-stable">stable</div>
+              </h4>
+            </div>
+            <div class="clear"></div>
+            <h3 class="version">{{ version }}</h3>
+            <h6 class="version-date">2020-09-24</h6>
+            <div class="description">Align code with Quarkus 1.9</div>
```
Comment (gunnarmorling) [inline _includes/release.html:60]: See above.

File: _includes/release.html [hunk 3]
```diff
@@ -1,157 +1,188 @@
...
+      <td>Java</td>
+      <td>8+</td>
+      <td>8+</td>
+    </tr>
+    <tr class="data">
+      <td>Kafka Connect</td>
+      <td>1.x, 2.x</td>
+      <td>1.x, 2.x</td>
+    </tr>
+    <tr class="data">
```
Comment (gunnarmorling) [inline _includes/release.html:92]: Should be obtained from data files, not hard-coded.

**Review by gunnarmorling — COMMENTED** [review]
Seems like there still are some hard-coded left-overs, @indraraj?

File: _includes/release.html (hunk 1 above)
Comment (mdrillin) [reply _includes/release.html:17]: @gunnarmorling - @indraraj was aware that there was still work to do - I PR'd some additional changes for him to review

Comment (gunnarmorling) [reply _includes/release.html:17]: Excellent, thx!

**Pull request merged**